src/test/resources/application.conf

```diff
@@ -1,6 +1,8 @@
 driver="org.postgresql.Driver"
-username="tdr"
-password="password"
-url="jdbc:postgresql://localhost:5432/consignmentapi"
+username="dGRy" # tdr
```

Comment:
Is it worth adding comment to explain what the string in the comment is?

Reply:
I've added in the same comment as the others.